nixos/modules/services/ttys/kmscon.nix

```diff
@@ -32,6 +32,13 @@ in {
         default = "";
         example = "font-size=14";
       };
+
+      extraOptions = mkOption {
+        description = "Extra flags to pass to kmscon.";
+        type = types.string;
```

Comment:
So far I've seen `types.string`, `types.separatedString " "` and `types.listOf types.str`.
I can definitely change it, though.